encoding/ascii85_test.ts

```diff
@@ -204,7 +206,7 @@ Deno.test({
 
     for (const [input, expect] of tests) {
       assertEquals(
-        decodeAscii85(input),
+        decodeAscii85(input as string),
```

Comment:
This change is unnecessary if the `tests` variable is declared using `as const` (`const` assertion). This fact should be checked whenever a non-changing variable is used.